### PR TITLE
doc: updated the dashboard feature list

### PR DESCRIPTION
**Pull request by LenzGr**
Updated and improved the Mgr Dashboard feature list: added RBD management features.

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

### CODE REVIEW

**Comment by LenzGr** [comment]
jenkins render docs

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -35,12 +35,16 @@ The dashboard plugin is implemented as a web application that visualizes
...
+The dashboard currently provides the following features to monitor and manage
+various aspects of your Ceph cluster:
 
+* **Username/password protection**: The Dashboard can only be accessed by
+  providing a configurable username and password.
 * **Overall cluster health**: Displays the overall cluster status, storage
   utilization (e.g. number of objects, raw capacity, usage per pool), a list of
-  pools and their status and usage statistics, access to the cluster log file.
+  pools and their status and usage statistics.
+* **Cluster logs**: Diplay the latest updates to the cluster's event and audit
```
Comment (s0nea) [inline doc/mgr/dashboard.rst]: Just a little typo: "Display"

Comment (LenzGr) [reply doc/mgr/dashboard.rst]: Fixed, thank you!

**Comment by LenzGr** [comment]
jenkins render docs

**Pull request merged**